app/build.gradle

```diff
@@ -4,20 +4,27 @@ plugins {
     id 'com.google.gms.google-services'
 }
 
+apply from: 'dependencies.gradle'
+def secretsPropertiesFile = rootProject.file("secrets.properties")
+def secretsProperties = new Properties()
+secretsProperties.load(new FileInputStream(secretsPropertiesFile))
+
 android {
-    compileSdk 31
+    compileSdk versions.compileSdk
 
     defaultConfig {
-        applicationId "com.cornellappdev.scoop"
```

Comment:
can you also change the repo name?

Reply:
I think someone with admin privilege on the repo can change.